roles/installer/templates/deployments/task.yaml.j2

```diff
@@ -340,6 +238,8 @@ spec:
                 - /var/lib/pre-stop/scripts/termination-master
 {% endif %}
           env:
+            - name: AWX_COMPONENT
```

Comment:
this is added so that awx get_task_qeuename() function can identify when in web container and will try to select a task instance's queue